From 7613c1104e0eb8626f3b199d58cc5821dca223dd Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:24:40 -0400
Subject: [tools] perf/lock: Remove dead code

Message-id: <1408462094-14194-112-git-send-email-jolsa@redhat.com>
Patchwork-id: 88139
O-Subject: [PATCH RHEL7.1 BZ1131394 111/325] perf lock: Remove dead code
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 02ad0702e54f9b82b697718e7e8662eb3f2266ee
Author: Davidlohr Bueso <davidlohr@hp.com>
Date: Sun Sep 8 19:19:13 2013 -0700

description
===========
No need for break statements after goto jumps.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-lock.c b/tools/perf/builtin-lock.c
index ee33ba2..148f7e2 100644
--- a/tools/perf/builtin-lock.c
+++ b/tools/perf/builtin-lock.c
@@ -446,7 +446,6 @@ broken:
   list_del(&seq->list);
   free(seq);
   goto end;
-  break;
  default:
   BUG_ON("Unknown state of lock sequence found!\n");
   break;
@@ -508,8 +507,6 @@ static int report_lock_acquired_event(struct perf_evsel *evsel,
   list_del(&seq->list);
   free(seq);
   goto end;
-  break;
-
  default:
   BUG_ON("Unknown state of lock sequence found!\n");
   break;
@@ -564,7 +561,6 @@ static int report_lock_contended_event(struct perf_evsel *evsel,
   list_del(&seq->list);
   free(seq);
   goto end;
-  break;
  default:
   BUG_ON("Unknown state of lock sequence found!\n");
   break;
@@ -606,7 +602,6 @@ static int report_lock_release_event(struct perf_evsel *evsel,
  switch (seq->state) {
  case SEQ_STATE_UNINITIALIZED:
   goto end;
-  break;
  case SEQ_STATE_ACQUIRED:
   break;
  case SEQ_STATE_READ_ACQUIRED:
@@ -624,7 +619,6 @@ static int report_lock_release_event(struct perf_evsel *evsel,
   ls->discard = 1;
   bad_hist[BROKEN_RELEASE]++;
   goto free_seq;
-  break;
  default:
   BUG_ON("Unknown state of lock sequence found!\n");
   break;
-- 
1.7.1